src/utils.spec.tsx

```diff
@@ -244,6 +245,44 @@ describe(__filename, () => {
       );
     });
   });
+
+  describe('makeReviewersURL', () => {
+    it.each(['apiHost', 'reviewersHost'])(
```

Comment:
Why would `apiHost` be involved in `makeReviewersURL`?

Reply:
Because `makeReviewersURL` replaces `apiHost` with `reviewersHost`, but I see your suggestion below to change that, and changing that will remove the need to worry about `apiHost` for the tests.